Formula/o/opensca-cli.rb

```diff
@@ -0,0 +1,19 @@
+class OpenscaCli < Formula
+  desc "OpenSCA is a supply-chain security tool for security researchers and developers"
+  homepage "https://opensca.xmirror.cn"
+  url "https://github.com/XmirrorSecurity/OpenSCA-cli.git", tag: "v3.0.0", revision: "f5e43fcd2fdbf2de9c5326f01f0395738dfec1b8"
```

Comment:
```suggestion
  url "https://github.com/XmirrorSecurity/OpenSCA-cli.git", 
      tag: "v3.0.0", 
      revision: "f5e43fcd2fdbf2de9c5326f01f0395738dfec1b8"
```

Reply:
Thank you for your patient explanation. I have resubmitted as instructed.